### ISSUE TITLE
Remove redirect banner

### ISSUE THREAD

**Issue by madanalogy** [TEAMMATES/teammates#10780]
Banner was set up in #10471 to accommodate the transition from v6 to v7. It is now no longer necessary with the end of life for v6 as of [V7.3.0](https://github.com/TEAMMATES/teammates/releases/tag/V7.3.0).

**Comment by damithc** [comment]
On a related note, it would be good to have a way to add a banner easily. e.g., if we want to quickly add a banner to convey some info to users (`currently facing some performance issues ... ` etc.)

**Comment by madanalogy** [comment]
> On a related note, it would be good to have a way to add a banner easily. e.g., if we want to quickly add a banner to convey some info to users (`currently facing some performance issues ... ` etc.)

Would you rather we just refactor the banner instead? So what will be displayed depends on input in the `config.ts`. Could be configured to allow for links etc, and wouldn't show if that attribute in the config is blank

**Comment by damithc** [comment]
> Would you rather we just refactor the banner instead? So what will be displayed depends on input in the `config.ts`. Could be configured to allow for links etc, and wouldn't show if that attribute in the config is blank

One idea: config.ts (or some other suitable config file) will have two variables, one to point to a banner to be shown to instructors, and one for students. The variable will actually point to a file containing the banner content. If the variable is empty, we don't show any banner.

**Comment by wkurniawan07** [comment]
> One idea: config.ts (or some other suitable config file) will have two variables, one to point to a banner to be shown to instructors, and one for students. The variable will actually point to a file containing the banner content. If the variable is empty, we don't show any banner.

Suggest to defer this to another ticket. This issue is strictly about removing the existing redirect banner.
The new suggestion requires very careful design, otherwise it could end up like the old MOTD feature.

**Comment by damithc** [comment]
> Suggest to defer this to another ticket. This issue is strictly about removing the existing redirect banner.
> The new suggestion requires very careful design, otherwise it could end up like the old MOTD feature.

Yes, should be a separate issue. And not anything urgent either.
